pkgs/development/python-modules/marisa/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, buildPythonPackage, fetchFromGitHub, marisa, swig }:
+
+buildPythonPackage rec {
+  pname = "marisa";
+  version = "1.3.40";
+
+  src = fetchFromGitHub {
+    owner = "s-yata";
+    repo  = "marisa-trie";
+    rev   = "59e410597981475bae94d9d9eb252c1d9790dc2f";
+    sha256 = "0z4bf55np08q3cbi6gvj3cpw3zp8kf2d0jq6k74pjk066m7rapbb";
+  };
+
+  nativeBuildInputs = [ swig marisa ];
+  buildinputs = [ marisa ];
+
+  sourceRoot = "${src.name}/bindings/python";
+
+  meta = with stdenv.lib; {
+    description = "Python binding for marisa package (do not confuse with marisa-trie python bindings)";
```

Comment:
yet the homepage does point to marisa-trie ?

Reply:
Well, as I mentioned it previously (https://github.com/NixOS/nixpkgs/pull/34416#discussion_r164831390) there are two different bindings for marisa-trie C++ library: one are generated by swig (this package), so the homepage of bindings is the homepage of C++ library, other bindings (also called marisa-trie) are from the different package (you can find them here  https://github.com/pytries/marisa-trie/). However, they are incompatible.

At first I packaged the bindings by pytries and called the package `marisa-trie`, but then I'd found out that I need bindings from s-yata's repo, so I'd decided to give latter the name as for the C++ package.

I agree that the description is pretty misleading, do you have any suggestions for improvement?